src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
+const seven = 7;
+const three = 3;
+const saleForThreeDays = 20;
+const saleForSevenDays = 50;
+const startPrice = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= seven) {
+    return days * startPrice - saleForSevenDays;
+  } else if (days >= three) {
+    return days * startPrice - saleForThreeDays;
+  }
+
+  return 80;
```

Comment:
The return value here is hard-coded, which is not a good practice. It should be calculated based on the `days` parameter and `startPrice` constant. Also, the return value here should be `days * startPrice`, not `80`.